dev/langtool/meta/ckeditor.core/meta.txt

```diff
@@ -69,7 +69,7 @@ common.invalidCssLength = Error message displayed to the user when the specified
 common.invalidHtmlLength = Error message displayed to the user when the specified length value is not a valid HTML measurement unit.
 common.invalidInlineStyle = Error message displayed to the user when the specified style value is not a valid CSS declaration.
 common.cssLengthTooltip = Tooltip displayed for the Width and Height fields in the Table Properties dialog window explaining the correct format of these length values.
-common.unavailable =
+common.unavailable = 
```

Comment:
Me neither. ;) I just use `dev/langtool/langtool.sh` and it was somehow affected by this.